mne/datasets/hf_sef/hf_sef.py

```diff
@@ -16,7 +16,8 @@ def data_path(dataset='evoked', path=None, force_update=False,
               update_path=True, verbose=None):
     u"""Get path to local copy of the high frequency SEF dataset.
 
-    Gets a local copy of the high frequency SEF MEG dataset [1]_.
+    Gets a local copy of the high frequency SEF MEG dataset
+    :footcite: `NurminenEtAl2017`.
```

Comment:
there is a space here that shouldn't be there:
```suggestion
    :footcite:`NurminenEtAl2017`.
```